app/frontend/src/components/designer/FormViewer.vue

```diff
@@ -61,46 +62,48 @@ export default {
         const response = await formService.getSubmission(this.submissionId);
         this.submissionRecord = Object.assign({}, response.data.submission);
         this.submission = this.submissionRecord.submission;
-        // simulate getFormName()
         this.formName = response.data.form.name;
-        // simulate getFormSchema()
         this.formSchema = response.data.version.schema;
-        // this.version = response.data.version.version;
+        this.version = response.data.version.version;
       } catch (error) {
         this.addNotification({
           message: 'An error occurred fetching the submission for this form',
           consoleError: `Error loading form submission data ${this.submissionId}: ${error}`,
         });
       }
     },
-    // Get form name
-    async getFormName() {
-      try {
-        const response = await formService.readForm(this.formId);
-        if (!response.data) {
-          throw new Error(`No data in response. FormId: ${this.formId}`);
-        }
-        this.formName = response.data.name;
-      } catch (error) {
-        this.addNotification({
-          message: 'An error occurred fetching this form',
-          consoleError: `Error loading form ${this.formId}: ${error}`,
-        });
-      }
-    },
     // Get the form definition/schema
     async getFormSchema() {
       try {
-        const response = await formService.readVersion(
-          this.formId,
-          this.versionId
-        );
-        if (!response.data || !response.data.schema) {
-          throw new Error(
-            `No schema in response. VersionId: ${this.versionId}`
-          );
+        let response = undefined;
```

Comment:
It doesn't look like we use the response variable outside of the if/else block. In this case, it's better to declare const response within the if/else blocks instead as it would be a tighter scope.

Reply:
Ah yeah, leftover from earlier attempt. Fixed